src/data/interfaces/AccidentData.ts

```diff
@@ -12,9 +14,11 @@ export type AccidentData = {
   Proces: string; // reason of the accident
   Beschrijving: string; // description
   Melder: string; // reporter
-  "Eerste tijd ter plaatse": Date,
-  "Laatste eindtijd": Date,
-  ovd: Date,
+  "Eerste tijd ter plaatse": Date;
+  "Laatste eindtijd": Date;
+  ovd: Date;
+  Latitude: number;
```

Comment:
What is the difference between Latitude and Latitude_van?

Reply:
Again, it's because of the differences between Brabant and RWS. In RWS, the fields are simply Latitude and Longiture, while in Brabant they are Latitude_van, Longitude_van (van=from), and possibly also Latitude_tot and Longitude_tot (tot=to) if the accident is a LineString. I decided to keep all these fields so the data is still compatible with both Brabant and RWS, but what I hope is that, if this project continues as more than just a demo, we will move the data processing on the backend and these inconsistencies will be solved.